openjdk/excludes/ProblemList_openjdk8-openj9.txt

```diff
@@ -290,6 +290,7 @@ sun/security/ssl/X509TrustManagerImpl/distrust/Entrust.java https://github.com/a
 sun/security/ssl/X509TrustManagerImpl/distrust/Symantec.java https://github.com/eclipse-openj9/openj9/issues/21380 generic-all
 sun/security/tools/jarsigner/diffend.sh	https://github.com/adoptium/aqa-tests/issues/125	linux-all
 sun/security/tools/jarsigner/emptymanifest.sh	https://github.com/adoptium/aqa-tests/issues/125	generic-all
+sun/security/tools/jarsigner/TimestampCheck.java https://bugs.openjdk.org/browse/JDK-8352302 generic-all
```

Comment:
This shouldn't be needed; see https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/809.

Reply:
Created https://github.com/adoptium/aqa-tests/pull/6122

Reply:
Sorry, didn't noticed that.